libs/accounts/src/lib/accounts-manager.tsx

```diff
@@ -60,13 +81,108 @@ export const AccountsManager = ({ partyId }: AccountsManagerProps) => {
     },
     [gridRef]
   );
+
   const { data, error, loading } = useDataProvider<
     AccountFieldsFragment[],
     AccountFieldsFragment
   >({ dataProvider: accountsDataProvider, update, variables });
+  const [depositDialog, setDepositDialog] = useState(false);
+
+  const symbols = data && getSymbols(data);
+  return (
+    <Web3Container>
+      <AsyncRenderer loading={loading} error={error} data={assetSymbols}>
+        {symbols &&
+          symbols.map((assetSymbol, i) => (
+            <AssetAccountTable
+              ref={gridRef}
+              key={assetSymbol}
+              assetSymbol={assetSymbol}
+              data={data}
+              hideHeader={i !== 0}
+            />
+          ))}
+      </AsyncRenderer>
+      <div className="m-auto ml-4">
+        <Button size="sm" onClick={() => setDepositDialog(true)}>
+          {t('Deposit new asset')}
+        </Button>
+      </div>
+      <DepositDialog
```

Comment:
chore (blocking): Lets try and keep lib components/functionality separated. Can you move the deposits dialog and state to be rendered by the app only. 

I suggest moving libs/accounts/src/lib/accounts-container.tsx to apps/trading/pages/portfolio (see apps/trading/pages/portfolio/withdrawals-container)